src/types/types.ts

```diff
@@ -99,9 +133,19 @@ export interface MediaItemType {
 }
 
 export interface LocationItemType {
-  id?: string
-  title?: string
-  to?: string
+  id: string
+  image: MediaItemType
+  locationType: string
+  title: string
+  to: string
+  uri: string
+  affiliateLibraryUrl: string
+  address: string
+  addressLink: string
+  amenities: AmenitiesType[]
+  reserveSeat: string
+  isUclaLibrary: boolean
+  libcalLocationIdForHours: string
```

Comment:
Since this is so similar to the LocationItemType in SectionLocationList.vue, I think it would make sense to collapse these 2 types together so that we don't confuse ourselves later. 

Can we revise it to look like 

`  
  id?: string
  image: MediaItemType
  locationType: string
  title: string
  to: string
  uri?: string
  affiliateLibraryUrl: string
  address: string
  addressLink: string
  amenities: AmenitiesType[]
  reserveSeat: string
  isUclaLibrary: boolean
  libcalLocationIdForHours: string
`
and then delete the version in SectionLocationList.vue and import from the types file instead? 

`import type { AmenitiesType, MediaItemType, LocationItemType } from '@/types/types'`